### PR TITLE
I changed some things.

### PR DESCRIPTION
**Pull request by PoweredByCoconut**
Stumbled across your repo yesterday, and thought it was really cool. A lot of files had direct references to files like "/home/todd/programs/...". I modified it to not use only relative paths and also have module for the backend source code so it doesn't have to be downloaded separately. I edited the readme to have installation instructions as well. Other than that I moved some things around to my liking and changed any folders or files to remove spaces as they can be annoying when typing in a terminal. I really like what you have, great job. Don't think you have to include what I changed, if you like it how it is, that's fine, you do you.

EDIT:
also if you want me to change some things and resubmit the pull request, I can do that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,32 @@
...
+### Install
+```
+git clone https://github.com/DJPretzel-bit64/acer-predator-rgb-gui --recursive
+cd acer-predator-rgb-gui/acer-predator-turbo-and-rgb-keyboard-linux-module
+chmod +x ./*.sh
+sudo ./install_service.sh
+```
+
+### Run
+Open the "rgb Keyboard GUI" folder and run "RGB Control.py". To activate the custom profile, run
```
Comment (ample-samples) [inline README.md]: This line looks to be incomplete, is this intentional?

**Review by ample-samples — COMMENTED** [review]
Nice work, I had intended to distribute this but had abandoned the project a while ago as I no longer need it but it's nice to see it refreshed 🙂 
Just one comment for you to look at

File: README.md (hunk 1 above)
Comment (PoweredByCoconut) [reply README.md]: Oh, yeah. I was writing that out and got distracted. I'll fix that.

File: README.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ This application was created to help users control their keyboard lights in a us
 ## Installation
 
 ### Dependencies
-The GUI requires Linux Headers, Python, Tkinter and CustomTkinter to run.
```
Comment (ample-samples) [inline README.md]: If linux-headers is in fact a dependency, it should be stated that it's kernel specific (users should install headers for their kernel, e.g. Zen kernel -> `sudo pacman -S linux-zen-headers`)

**Review by ample-samples — COMMENTED** [review]
The speed slider is a nice edition, just one comment for you

File: README.md (hunk 2 above)
Comment (PoweredByCoconut) [reply README.md]: oh, ok. I'll add that quickly

**Comment by ample-samples** [comment]
Nice one! Merged 👍

**Pull request merged**